CMakeLists.txt

```diff
@@ -0,0 +1,211 @@
+# Copyright (c) 2020 ARM Limited. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+# This is the boilerplate for Mbed OS
+
+cmake_minimum_required(VERSION 3.19.0 FATAL_ERROR)
+
+# Using relative paths behavior
+if(POLICY CMP0076)
```

Comment:
I'll create a ticket, we got some relative paths and somewhere using source dir variable. I recall we added this long time ago to suppress Cmake warning. We'll test with the current code base, to check if this can be removed.

Reply:
IOTOSM-2643 created